__tests__/app.test.js

```diff
@@ -162,5 +161,35 @@ describe("app", () => {
           expect(body.comments).toHaveLength(0);
         });
     });
+    test("201: POST - adds an object onto comments with the properties username and body and responds with the posted comment", () => {
+      const newComment = {
+        username: "HappyJuice123",
+        body: "i like umbrellas",
+      };
+
+      return request(app)
+        .post("/api/reviews/1/comments")
+        .send(newComment)
+        .expect(201)
+        .then(({ body }) => {
+          const { addComment } = body;
+          expect(addComment).toMatchObject({
+            comment_id: 7,
+            body: "i like umbrellas",
+            votes: expect.any(Number),
+            author: "HappyJuice123",
+            review_id: 1,
+            created_at: expect.any(String),
+          });
+        });
+    });
+    test("404: responds with msg when sent a query with a valid but non-existent review_id", () => {
```

Comment:
Good test, how about for when:
the user tries to enter a string user_id?
the user doesn't exist?
missing fields on request body?
additional fields on request body?